docarray/array/mixins/evaluation.py

```diff
@@ -14,31 +14,64 @@ class EvaluationMixin:
 
     def evaluate(
         self,
-        other: 'DocumentArray',
         metric: Union[str, Callable[..., float]],
+        ground_truth: Optional['DocumentArray'] = None,
```

Comment:
It is a breaking change anyway, because we need to turn it from a mandatory into an optional attribute, but I can change it back to `other` if you think it makes a difference. I just find this name confusion, especially as an keyword argument.

Reply:
it is not breaking, passing from mandatory to Optional will not break other's code, it just adds more options